db/seeds.rb

```diff
@@ -17,6 +17,7 @@
 users = [
   { email: "alex.lee@education.gov.uk", family_name: "Alex", given_name: "Lee" },
   { email: "alisa.ali@education.gov.uk", family_name: "Ali", given_name: "Alisa" },
+  { email: "brandon.chan@education.gov.uk", family_name: "Chan", given_name: "Brandon" },
```

Comment:
I think Brandon email is: `brandon1.chan@education.gov.uk`. It shows like this on Slack, Outlook profile...

Reply:
Correct, I have brandon1.chan@education.gov.uk